.github/workflows/release.yaml

```diff
@@ -7,14 +7,44 @@ on:
       - 'next'
 
 jobs:
-  kubernetes-version-tests:
+  latest-tests:
```

Comment:
There is deletion/adding confusing between this and #1618 

Actionable: can you please combine this change along with #1618 ?

Reply:
Ok.

Reply:
I'll merge this in, and then update #1618 to adjust for it before that merges :+1: